dev-docs/modules/consentManagementGpp.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: page_v2
+page_type: module
+title: Consent Management - GPP
+description: If you have users in regions of the world that adhere to the Global Privacy Platform, this module works with your Consent Management Platform to pass consent info to bidders and help align with regional regulations.
+module_code : consentManagementGpp
+display_name : Consent Management - GPP
+enable_download : true
+recommended: true
+sidebarType : 1
+---
+
+# GPP Consent Management Module
+{: .no_toc }
+
+* TOC
+{: toc }
+
+{% capture legalNotice %}
+  This resource should not be construed as legal advice and Prebid.org makes no guarantees about compliance with any law or regulation. Please note that because every company and its collection, use, and storage of personal data is different, you should seek independent legal advice relating to obligations under European, Canadian and /or US regulations, including the GDPR, the ePrivacy Directive and CCPA. Only a lawyer can provide you with legal advice specifically tailored to your situation. Nothing in this guide is intended to provide you with, or should be used as a substitute for, legal advice tailored to your business.
```

Comment:
clarify the wording of "Please note that because every company and its collection, use, and storage of personal data is different, you sho.." to something like, "Please note that because every compan**ies** collection, use, and storage of personal data is different, you sho

Reply:
Totally agree with @jdwieland8282. Only small amendment would be that is should be "company's" instead of "companies"

Reply:
Thanks for the feedback, I pushed a change with some updated wording.